packages/cf-util-http/package.json

```diff
@@ -12,6 +12,10 @@
   },
   "dependencies": {
     "cf-util-logger": "^2.2.0",
-    "superagent": "^1.4.0"
-  }
+    "isomorphic-fetch": "^2.2.1"
+  },
+  "optionalDependencies": {
+    "es6-promise": "^4.1.0"
```

Comment:
It's optional and doesn't get installed by default. Only needed for whoever uses this package on an environment that doesn't have a spec-compliant Promise, like node v4. It's a dependency of isomorphic-fetch.